run_test.sh

```diff
@@ -50,11 +50,7 @@ echo "cd $PWD"
 # Clone PICSAR and AMReX
 git clone --branch development https://github.com/AMReX-Codes/amrex.git
 # Use QED brach for QED tests
-if [ "${WARPX_CI_QED}" = "TRUE" ]; then
-    git clone --branch development https://github.com/ECP-WarpX/picsar.git
-else
-    git clone --branch development https://github.com/ECP-WarpX/picsar.git
-fi
+git clone --branch improve_makefile https://github.com/lucafedeli88/picsar.git
```

Comment:
Memo to ourselves: let's not forget to wait for the upstream PR to be merged and then revert this change here